oc-chef-pedant/lib/pedant/platform.rb

```diff
@@ -195,18 +195,18 @@ def create_client(name, org = self.test_org)
       payload = { "name" => clientname }
 
       if server_api_version == 0
-        r = post(api_url('/clients'), org.validator, :payload => payload)
+        r = post(api_url('/clients', org), org.validator, :payload => payload)
```

Comment:
This is just for testing, so no, but also I'm fixing a bug. This code worked in the default case (where you didn't pass in an org) that was mostly being used but was broken if you passed an org in. Just fixing a bug.